create/how_to/updating.md

```diff
@@ -0,0 +1,314 @@
+
+# execute the server binary
+./target/release/meilisearch
+```
+
+:::
+
+::: tab APT
+
+Replace `0.X.X` with the version you would like to install.
+
+```bash
+apt-get install meilisearch=0.X.X
+```
+
+:::
+
+::: tab Docker
+
+Replace `0.X.X` with the version you would like to install.
+
+```bash
+docker run -it --rm \
+    -p 7700:7700 \
+    -v $(pwd)/data.ms:/data.ms \
+    getmeili/meilisearch:v0.X.X
+```
+
+:::
+
+::::
+
+### Proceed according to your database version
+
+Now that you know which MeiliSearch version your database is compatible with, proceed accordingly:
+
+- **If your database version is v0.15.0 or above**, continue to the [next section](#updating-from-v0-15-0-or-above).
+- **If your version is v0.14.0 or below**, [go here](#updating-from-v0-14-0-or-below).
+
+## Updating from v0.15.0 or above
+
+Because MeiliSearch v0.15.0 and above include the [dumps feature](/reference/features/dumps.md), updating is very simple.
+
+You just need to:
+
+1. [Set all fields as displayed attributes](#step-1-set-all-fields-as-displayed-attributes)
+2. [Create a dump using the MeiliSearch version that's compatible with your database](#step-2-create-the-dump)
+3. [Import the dump using the most recent MeiliSearch version](#step-3-import-the-dump)
+
+### Step 1: Set all fields as displayed attributes
+
+When creating dumps, MeiliSearch calls the same method as the [get documents endpoint](/reference/api/documents.md#get-documents). This means that all fields must be [displayed](/reference/features/field_properties.md#displayed-fields) in order to be saved in the dump.
+
+Start by using the [get displayed attributes endpoint](/reference/api/displayed_attributes.md#get-displayed-attributes) to verify that **all attributes are displayed**.
+
+```bash
+# whenever you see :index_uid, replace it with your index's unique id
+curl -X GET \
+  'http://127.0.0.1:7700/indexes/:index_uid/settings/displayed-attributes'
+```
+
+If the response is `{'displayedAttributes': '["*"]'}`, you can move on to the [next step](#step-2-create-the-dump).
+
+If it's something else, then you need to use the [reset displayed attributes endpoint](/reference/api/displayed_attributes.md#reset-displayed-attributes). Before doing this, make sure you save your list of displayed attributes somewhere so you can restore it afterwards.
+
+```bash
+curl -X DELETE \
+  'http://127.0.0.1:7700/indexes/:index_uid/settings/displayed-attributes'
+```
+
+This command should return an updateId:
+
+```json
+{
+  "updateId": 1
+}
+```
+
+Now that all fields are displayed, proceed to the next step.
+
+### Step 2: Create the dump
+
+Before creating your dump, make sure that your [dump directory](/reference/features/configuration.md#dumps-destination) is somewhere accessible. By default, dumps are created in a folder called `dumps` at the root of your MeiliSearch directory.
+
+If you're unsure where your MeiliSearch directory is located, try this:
+
+:::: tabs
+
+::: tab OSX / Linux
```

Comment:
`OSX` isn't the name anymore. They changed for `macOS` so we should changed it here, and elsewhere in the documentation.

Reply:
Didn't realize this! I feel kind of old now.